privacyidea/lib/token.py

```diff
@@ -2092,6 +2092,7 @@ def create_challenges_from_tokens(token_list, reply_dict, options=None):
             message_list.append(message)
             if r_chal:
                 challenge_info = challenge_info or {}
+                challenge_info["challenge_type"] = "generic_challenge"
```

Comment:
please add the `generic_challenge` to the `CHALLENGE_TYPE` class here:
https://github.com/privacyidea/privacyidea/blob/28f138522618915950769a0fe196720f3b417393/privacyidea/lib/challengeresponsedecorators.py#L46